src/slider/__tests__/utils.test.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { getPercent, getStepArray, findLowerAndHigherVals } from '../../../lib/components/slider/utils';
+
+describe('getPercent', () => {
+  test('works with different min value', () => {
+    expect(getPercent(100, 50, 150)).toEqual(50);
+  });
+
+  test('works with negative values', () => {
+    expect(getPercent(0, -100, 100)).toEqual(50);
+  });
+});
+
+describe('getStepArray', () => {
+  test('works with indivisible step', () => {
+    expect(getStepArray(6, 0, 20)).toEqual([0, 6, 12, 18]);
+  });
+
+  test('works with negative numbers', () => {
+    expect(getStepArray(5, -10, 10)).toEqual([-10, -5, 0, 5, 10]);
+  });
+});
+
+describe('findLowerAndHigherVals', () => {
```

Comment:
I would add two more cases: when there is no lower value and when there is no higher value.

Reply:
Good catch.